### PR TITLE
add GetRealAddress & coverage badge

### LINKED ISSUE #11: Add support for Street Address
would be great to generate street addresses something like:

```
419 Indian Spring Street
Hattiesburg, MS 39401
```

could just be US addresses for now, but would be great to also specify regions and get random addresses based on different regions like the UK/Europe/China etc

### PR DESCRIPTION
**Pull request by wolf-joe**
fix  #11

### CODE REVIEW

**Review by adityarizkyramadhan — APPROVED** [review]

**Review by geshtng — APPROVED** [review]

File: address.go [hunk 1]
```diff
@@ -1,11 +1,28 @@
 package faker
 
 import (
+	_ "embed"
+	"encoding/json"
+	"github.com/go-faker/faker/v4/pkg/options"
 	"reflect"
```
Comment (geshtng) [inline address.go:7]: Let's separate import to be tidier
1. Go's package
2. External package
3. Internal package
```suggestion
	_ "embed"
	"encoding/json"
	"reflect"
	
	"github.com/go-faker/faker/v4/pkg/options"
```

Comment (wolf-joe) [reply address.go]: using gofmt solve this

**Review by geshtng — APPROVED** [review]

File: address.go [hunk 2]
```diff
@@ -64,3 +82,24 @@ func Latitude(opts ...options.OptionFunc) float64 {
...
 }
+
+type RealCoordinates struct {
+	Latitude  float64 `json:"lat"`
+	Longitude float64 `json:"lng"`
+}
+
+type RealAddress struct {
+	Address1    string          `json:"address1"`
+	Address2    string          `json:"address2"`
```
Comment (bxcodec) [inline address.go:93]: What are this address1 and address2 represent to?
Do we have a better naming?

Comment (wolf-joe) [reply address.go:93]: `address1` and `address2` are copy from original json file.
looks like `address1` is just normal address, `address2` is empty in mostly case.
I will remove `address2` just now.

File: address.go [hunk 3]
```diff
@@ -64,3 +82,23 @@ func Latitude(opts ...options.OptionFunc) float64 {
...
 	}, opts...).(float64)
 }
+
+type RealCoordinates struct {
+	Latitude  float64 `json:"lat"`
+	Longitude float64 `json:"lng"`
+}
+
+type RealAddress struct {
+	Address     string          `json:"address1"`
```
Comment (bxcodec) [inline address.go:92]: the JSON tag still address1

Comment (bxcodec) [reply address.go:92]: should we rename it @wolf-joe ?

**Pull request merged**